app/helpers/protocols_io_helper.rb

```diff
@@ -146,10 +150,16 @@ def close_open_html_tags(text)
     Nokogiri::HTML::DocumentFragment.parse(text).to_html
   end
 
+  def step_hash_null?(step_json)
+    return true if step_json.dig(0, 'components', 0, 'component_type_id').nil?
```

Comment:
`return true` and `false` is redundant.
Use only `step_json.dig(0, 'components', 0, 'component_type_id').nil?`